torchrl/_utils.py

```diff
@@ -176,3 +179,63 @@ def __repr__(self):
 
 
 _CKPT_BACKEND = _Dynamic_CKPT_BACKEND()
+
+
+class implement_for:
```

Comment:
Can we add this to the doc under `docs/source/`? We may need to create a special `.rst` file for the top-level utils.

Reply:
Added `implement_for` to the doc. @vmoens let me know if you want to add other methods from _utils.py to the documentation.